docs/howto/test_effects.md

```diff
@@ -407,4 +407,46 @@ object MySpec
           @@ timeout(20.nanos) //it's a good idea to compose `eventually` with `timeout`, or the test may never end
       ) @@ timeout(60.seconds)   //apply a timeout to the whole suite
     )
-``` 
\ No newline at end of file
+``` 
+
+#### @@ nonFlaky
```

Comment:
Great documentation. Can we add something about how we can't just create extra copies of arbitrary environment types? This does a great job of explaining the semantics but I think risks leaving the impression that the user has to do this just because we didn't do the work on our end to reset things rather than providing a fresh environment being conceptually something we can't do for a polymorphic environment type.

Reply:
good idea - wouldn't want to be accused of being lazy. On the topic of documentation, I've sort of rationalised the clock time vs fiber time issue in my head as vectors, with clock time being on the x axis and fibertime on the y axis, and the area above the identity function = sleepytime. I think it might be useful to others in quickly understanding how they interact. Here's a quick sketch of the idea:

![image](https://user-images.githubusercontent.com/30517573/67614360-77ce9e80-f807-11e9-9bcc-bcc5a219d9ba.png)

I haven't seen any graphics in the doco - is there a recommended approach/format/style for this sort of thing?

Reply:
Yes that is a great idea. I haven't seen a lot of graphics in this project either. Maybe someone on Discord could suggest a good solution?